### PR TITLE
Stylings

### PR DESCRIPTION
**Pull request by olanikegloria**
In this milestone, I learned to;
Understand the concept of state management.
Use store, actions and reducers in React.
Connect React and Redux.
Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+  .links-mobile {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .bookform {
+    display: flex;
+    flex-direction: column;
+  }
+}
```
Comment (usorfaitheloho) [inline src/App.css:300]: You did well with your bookstore app, kindly note that according to the design you have, it doesn't completely follow the design guidelines given. Please note that your design should be as close as possible to the design given 😄  so please redesign your app to be a replica of the design we have

Your design :

image

design given :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @olanikegloria  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by olanikegloria** [comment]
hello please I already added all the designs and this is the picture of the app
Screenshot (92)
Screenshot (93)

**Comment by olanikegloria** [comment]
I don't know why you have put a different picture of my app there but please can you help me check properly??

File: src/App.css (hunk 1 above)
Comment (peterrobert) [reply src/App.css:300]: This change has been implemented well

**Review by peterrobert — APPROVED** [review]
Hi @olanikegloria, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**